docs/tutorial-create-hello-world.md

```diff
@@ -1,3 +1,58 @@
 # Creating `hello-world.wasm`
 
-... more coming soon
+First, you'll need the standard rust toolchain.
+
+[follow these instructions to install `rustc`, `rustup` and `cargo`]: https://www.rust-lang.org/tools/install
```

Comment:
I think this syntax may not be what you want, instead of a `:` I think you'll want to put the url in parens